third_party/build/bin/bcc-build.sh

```diff
@@ -22,16 +22,45 @@ WORK_DIR=/tmp/build-"$PKGNAME"
 VERSION="$PKGVERSION-$ITERATION"
 
 function buildrequires() {
```

Comment:
Does this really work with `build.py` - my understanding is the file must be named `<dep>_build.sh` with an underscore. Same for `aioh2-build.sh`.